app/experimenter/experiments/tests/test_changelog_utils.py

```diff
@@ -142,8 +144,8 @@ def test_outputs_expected_schema_for_complete_experiment(self):
                 "risk_revenue": experiment.risk_revenue,
                 "secondary_outcomes": [secondary_outcome],
                 "slug": experiment.slug,
-                "status": experiment.status,
                 "status_next": experiment.status_next,
+                "status": experiment.status,
```

Comment:
another question did you manually changed the sequence of the variable or with formatter? 🤔

Reply:
Oh yeah I sorted the list alphabetically.